### PR TITLE
Improve [`send`/`bring`]`-to-`[`front`/`back`] metadata

### PR DESCRIPTION
**Pull request by danielbayley**
Following #1425…

### CODE REVIEW

File: icons/send-to-back.svg [hunk 1]
```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="14" y="14" width="8" height="8" rx="2" />
```
Comment (jguddas) [inline icons/send-to-back.svg:12]: I liked it better before.

Comment (danielbayley) [reply icons/send-to-back.svg:12]: > I liked it better before.

@jguddas 😅 Valid, but it isn’t consistent, or accurate… Maybe we ultimately decide that doesn’t matter? cc: @karsa-mistmere @ericfennis

Comment (ericfennis) [reply icons/send-to-back.svg:12]: I'm also with @jguddas other looked better. To me 100% it looks like two squares with connecting lines. The corners are tough to see in this adjustment.

Comment (karsa-mistmere) [reply icons/send-to-back.svg:12]: I also agree, my train of thought was the same as that of @ericfennis, while it's less consistent with its pair, the original design is a bit more legible.

Comment (danielbayley) [reply icons/send-to-back.svg:12]: > The corners are tough to see in this adjustment.

Another option would be to make them slightly sharper to compensate (as in third below)… Here are all the possible options for comparison:

preview

@jguddas @ericfennis @karsa-mistmere Thoughts?

Comment (jguddas) [reply icons/send-to-back.svg:12]: My vote is still for the original.

Comment (ericfennis) [reply icons/send-to-back.svg:12]: I vote for 1 and 3.

Comment (danielbayley) [reply icons/send-to-back.svg:12]: I think 3. So looks like @karsa-mistmere has the deciding vote…

Comment (karsa-mistmere) [reply icons/send-to-back.svg:12]: I'm also voting for the original — back when that icon was submitted I'd ruminated over posting a suggestion for no. 3 myself, so I definitely see where you're coming from, but I think all things considered, no. 1 simply has much better legibility.

We could check if alternatively, making `bring-to-front` more consistent with this one would work better (i.e. increasing the central square in that one).

Comment (karsa-mistmere) [reply icons/send-to-back.svg:12]: Column 1: current icons
Column 2: `bring-to-front` w/ larger rectangle to match `send-to-back`

image
image

Personally, I'm fine with either.

Comment (jguddas) [reply icons/send-to-back.svg:12]: I'm fine with making `bring-to-front` bigger.

Comment (danielbayley) [reply icons/send-to-back.svg]: > We could check if alternatively, making `bring-to-front` more consistent with this one (i.e. increasing the central square in that one) would work better.

I think that makes that one look much worse 😅 `reverted`, so just the metadata improvements then…

**Review by jguddas — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**